roles/sensu-check/handlers/main.yml

```diff
@@ -1,3 +1,5 @@
 ---
 - name: restart sensu-client missing ok
   service: name=sensu-client state=restarted must_exist=false
+  failed_when: false
```

Comment:
This is not the right way to handle a service restarting that is missing. We do not want to simply ignore the failure of the service to be restarted, as this does.